libs/designsystem/calendar/src/calendar.component.scss

```
@@ -48,17 +52,17 @@ td {
   align-items: center;
   justify-content: space-between;
 
-  button:disabled {
+  button[aria-disabled] {
```

I know we only apply this attribute when it's `true`, but we probably should include this in the selector as to not make it brittle for future changes: 
```suggestion
  button[aria-disabled="true"]  {
```